src/site/content/en/learn/pwa/windows/index.md

```diff
@@ -145,7 +145,12 @@ To reduce phishing within PWAs, some desktop browsers may have additional measur
 
 An experimental capability, known as *tabbed mode*​ will let your PWA have a tab-based design similar to a web browser. In this case, the user can have several tabs opened from the same PWA but all tied together in the same operating system window, as you can see in the following image:
 
-<video autoplay="" loop="" muted=""><source src="https://storage.googleapis.com/web-dev-uploads/video/8WbTDNrhLsU0El80frMBGE4eMCD3/JwN0btyXFGiT9oPFh2qJ.webm" type="video/webm"></video>
+{% Video src="video/W3z1f5ZkBJSgL1V1IfloTIctbIF3/I0idt5S5IvAu5iy0WCq7.mp4",
+autoplay="true",
+loop="true",
+muted="true",
+alt="Video showing the experimental tab-based design", width="800", height="519" %}
```

Comment:
You need to remove height and width for now, or you get a massive gap:

```suggestion
alt="Video showing the experimental tab-based design" %}
```

Waiting for #8399 to be merged to allow these to be used.

Reply:
Actually alt isn't valid on `video` either, so should remove that too.